waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -134,6 +135,15 @@ dbSeed genProjectDir seedName = do
     ExitSuccess -> Right ()
     ExitFailure c -> Left $ "Failed with exit code " <> show c
 
+dbIsRunning ::
+  Path' Abs (Dir ProjectRootDir) ->
+  IO Bool
+dbIsRunning genProjectDir = do
+  chan <- newChan
+  exitCode <- DbJobs.dbExecuteTest genProjectDir chan
+  -- TODO: should we inspect the error it gives?
```

Comment:
Ha good question. What do we do with it if we inspect it? I guess we would just propagate it all the way to user and say "hey, we failed to connect to the databse, and it returned this exit code".
So it depends, if you feel like there is a decent way to propagate this exit code back to user, we could go with that. If not and it feel unneccessary, then let's turn this TODO into a NOTE explaining our reasoning and it is fine.